cmake/common.cmake

```diff
@@ -605,10 +412,21 @@ set(CCF_NETWORK_TEST_ARGS
 )
 
 # SNIPPET: Lua generic application
-add_enclave_lib(luagenericenc ${CCF_DIR}/src/apps/luageneric/oe_sign.conf ${CCF_DIR}/src/apps/sample_key.pem SRCS ${CCF_DIR}/src/apps/luageneric/luageneric.cpp)
-
-add_enclave_lib(jsgenericenc ${CCF_DIR}/src/apps/jsgeneric/oe_sign.conf ${CCF_DIR}/src/apps/sample_key.pem SRCS ${CCF_DIR}/src/apps/jsgeneric/jsgeneric.cpp)
+add_enclave_lib(luagenericenc
```

Comment:
This will need the multi-line snippet comment to be displayed properly in the docs.

Reply:
Good catch - have updated the docs as well.